docs/index.md

```diff
@@ -4,17 +4,9 @@ TAU is the standard web UI library for Tizen platform. The acronym stands for Ti
 Advanced UI Library. It is a collection of UI controls called *widgets* which simplify
 application coding.
 
-## History
-
-TAU originated from Tizen Web UI Framework Library (standard library for Tizen 2.2.1),
-which was mainly based as an extension to jQuery Mobile. Key features of the former were
-coding simplification and app creation speed. With that in mind TAU was created as the
-framework *advanced to the next level*. TAU is a standalone UI library, without jQuery
-overhead, but that duo can be used as explained in this [guide later](using_jquery_with_tau.html).
-
 ## Benefits
 
-When using TAU you will get many benefits usable in your code.
+When using TAU you will get many benefits usable in your code:
 
 * A standalone library, meaning no additional libraries are needed
 * Can be used with jQuery, TAU exposes a special API to the jQuery object which is
```

Comment:
- The popularity of jQuery Mobile is currently small
- "ECMAScript5 and HTML5 compliant" at the moment it is rather required minimum